plugins/modules/kibana_alert.py

```diff
@@ -314,7 +337,11 @@ def main():
   kibana_alert = KibanaAlert(module)
   if state == 'present':
     if kibana_alert.alert:
-      results['msg'] = f'alert named {kibana_alert.alert_name} exists'
+      results['changed'] = True
```

Comment:
Need to do a comparison of the existing alert with the settings provided and only set `results['changed'] = True` if there are differences, otherwise it will always report `changed` even if there are no actual changes to make